modules/ROOT/pages/dataweave-formats-multipart.adoc

```diff
@@ -0,0 +1,306 @@
+= Multipart Format (Form Data)
+
+MIME Type: `multipart/form-data`
+
+ID: `multipart`
+
+DataWeave supports Multipart subtypes, in particular `form-data`. These formats
+enable you to handle several different data parts in a single payload,
+regardless of the format each part has. To distinguish the beginning and end of
+a part, a boundary is used and metadata for each part can be added through
+headers.
+
+DataWeeave represents a Multipart document with the given `Object` structure:
+
+[source,weave,linenums]
+----
+type Multipart = {
+  preamble?: String,
+  parts: {
+    _?: MultipartPart
+  }
+}
+type MultipartPart = {
+  headers?: {
+    "Content-Disposition"?: {
+      name: String,
+      filename?: String
+    },
+    "Content-Type"?: String
+  },
+  content: Any
+}
+----
+
+[[examples]]
+== Examples
+
+The following examples show uses of the Multipart format.
+
+* <<example1>>
+* <<example2>>
+* <<example3>>
+
+[[example1]]
+=== Example: Multipart
+
+This example shows how DataWeave reads simple Multipart content.
+
+==== Input
+
+The Multipart input serves as the payload to the DataWeave source.
+
+[source,text,linenums]
+----
+--34b21
+Content-Disposition: form-data; name="text"
+Content-Type: text/plain
+Book
+--34b21
+Content-Disposition: form-data; name="file1"; filename="a.json"
+Content-Type: application/json
+{
+"title": "Java 8 in Action",
+"author": "Mario Fusco",
+"year": 2014
+}
+--34b21
+Content-Disposition: form-data; name="file2"; filename="a.html"
+Content-Type: text/html
+
+<title> Available for download! </title>
+--34b21--
+----
+
+==== Source
+
+The DataWeave script transforms the Multipart input payload to the DataWeave (dw) format.
+
+[source,weave,linenums]
+----
+%dw 2.0
+output application/dw
+---
+payload
+----
+
+==== Output
+
+The output shows how the DataWeave (dw) format represents the Multipart input. Note that the `raw` and `content` values are  shortened for brevity. The full values are longer.
+
+[source,weave,linenums]
+{
```

Comment:
There's a missing block sterter here, this should be: 
[source,weave,linenums]
----
{

Reply:
There was a missing block demarcation